src/pages/CropSidebar/CropSidebar.js

```diff
@@ -146,20 +146,12 @@ const CropSidebar = ({
         const key = Object.keys(keyObject)[0];
         // get filter values ex. [1,2,3]
         const vals = keyObject[key];
-
-        // iterate over all crop.data categories
-        Object.keys(crop.data).forEach((category) => {
-          // check if crop.data[category] includes key
-          if (Object.keys(crop.data[category]).includes(key)) {
-            // make sure crop.data[category][key].values[0] exists
-            if (crop.data[category][key].values[0] !== undefined) {
-              // if there is not an intersection, match = false
-              if (!crop.data[category][key].values.some((item) => vals.includes(item))) {
-                match = false;
-              }
-            }
+        if (crop.attributes.filter((att) => att.label === key).length > 0) {
+          // if there is not an intersection, match = false
+          if (!crop.attributes.filter((att) => att.label === key)[0]?.values.some((item) => vals.includes(item))) {
+            match = false;
           }
-        });
+        }
       });
 
       cd[n].inactive = (!match)
```

Comment:
I got an error on this line when I'm testing in Kentucky:
`CropSidebar.js:157 Uncaught TypeError: Cannot read properties of undefined (reading 'Soil Conditions')
    at CropSidebar.js:157:1`